README.md

```diff
@@ -82,6 +82,14 @@ You can stop the server with CTRL + C.
 
 ### Testing
 
+Running tests requires that necessary composer packages, such as PHPUnit, are
+first installed for this repository. If you installed `core` using the 
+`base-distribution` package, then you must install the composer packages 
+separately. To do so, change to the base directory of `core` and run 
+`composer install`.
```

Comment:
Please don't. Developing the core package if it is installed as a dependency of the base-distribution package is a very bad idea as a composer update will overwrite any changes.

Reply:
The core package should only be changed if it is installed stand-alone, not if it is installed as a dependency.

Reply:
I'm perfectly fine with having documentation for running the tests. The documentation should not encourage people to modify dependencies, though (and maybe warn people to not do this).